src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -102,6 +105,66 @@ public override void ConfigureServices(IServiceCollection services)
             // Disabling same-site is required for OpenID's module prompt=none support to work correctly.
             services.ConfigureApplicationCookie(options => options.Cookie.SameSite = SameSiteMode.None);
         }
+
+        public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IServiceProvider serviceProvider)
+        {
+            var openIddictServerOptions = serviceProvider.GetRequiredService<IOptions<OpenIddictServerOptions>>().Value;
+
+            if (openIddictServerOptions.AuthorizationEndpointPath != PathString.Empty)
+            {
+                routes.MapAreaRoute(
+                    name: "Access.AuthorizeGet",
+                    areaName: OpenIdConstants.Features.Core,
+                    template: openIddictServerOptions.AuthorizationEndpointPath,
```

Comment:
I suspect this will result in a `PathString` -> `string` implicit conversion, that does some URL/percent-encoding under the hood.
If it's not desirable, you may want to use `PathString.Value` to get the unescaped path.